source/Ship.cpp

```diff
@@ -65,6 +65,7 @@ namespace {
 	const double MAXIMUM_TEMPERATURE = 100.;
 
 	const double SCAN_TIME = 600.;
+	const int MAX_SCAN_RANGE_FACTOR = 3 * 3; // three standard deviations AKA triple range
```

Comment:
```suggestion
	// Three standard deviations also known as triple range.
	const int MAX_SCAN_RANGE_FACTOR = 3 * 3;
```

Reply:
I added a longer comment explaining the whole thing.

Reply:
even better 👍